config/config.yaml.template

```diff
@@ -9,5 +9,9 @@ testing: #local db, local app
   host: localhost
   API port: 5050
 
+save_hosts:
+  local: blobs
```

Comment:
```suggestion
  local: <path/to/blobs>
```

Reply:
Made a lot of changes to how paths are stored in config file